system_tests/bigtable.py

```diff
@@ -93,29 +97,43 @@ def _retry_on_unavailable(exc):
 def setUpModule():
     from grpc._channel import _Rendezvous
     _helpers.PROJECT = TESTS_PROJECT
-    Config.CLIENT = Client(admin=True)
+    Config.IN_EMULATOR = os.getenv(BIGTABLE_EMULATOR) is not None
+
+    if Config.IN_EMULATOR:
+        credentials = EmulatorCreds()
+        Config.CLIENT = Client(credentials=credentials, admin=True)
+    else:
+        Config.CLIENT = Client(admin=True)
+
     Config.INSTANCE = Config.CLIENT.instance(INSTANCE_ID, LOCATION_ID)
-    retry = RetryErrors(_Rendezvous, error_predicate=_retry_on_unavailable)
-    instances, failed_locations = retry(Config.CLIENT.list_instances)()
 
-    if len(failed_locations) != 0:
-        raise ValueError('List instances failed in module set up.')
+    if not Config.IN_EMULATOR:
+        retry = RetryErrors(_Rendezvous, error_predicate=_retry_on_unavailable)
+        instances, failed_locations = retry(Config.CLIENT.list_instances)()
+
+        if len(failed_locations) != 0:
+            raise ValueError('List instances failed in module set up.')
```

Comment:
See the discussion on #2242. I actually agreed to drop the changes to the system test for this PR, thanks for the reminder.